raiden-dapp/package.json

```diff
@@ -18,7 +18,6 @@
     "serve:no-logging": "export NODE_ENV=nologging && vue-cli-service serve"
   },
   "dependencies": {
-    "core-js": "^3.8.3",
```

Comment:
Any ideas why this happened?

Reply:
Weird.. maybe some leftover from a `yarn install` or alike? You sure you started the process with a clean environment?

Reply:
Yeah, but the `prepare-release` script even runs `yarn install`

Reply:
Please, do a `serve` and also a prod `build` to check everything is fine without it. If yes, let it be.

Reply:
Ah, it's just duplicated in line https://github.com/raiden-network/light-client/pull/2535/files#diff-45d4a245bf7444d71c7e69b3bb96525d141aa212eeb829ddb643575ded0f9cfbR22